### PR TITLE
[WIP] Porting to ROS2

### PR DESCRIPTION
**Pull request by ipa-hsd**
- [ ] CI
- [ ] pf_interfaces
  - [ ] package.xml and CMakeLists.txt
- [ ] pf_description
  - [ ] package.xml and CMakeLists.txt
  - [ ] launch files
- [ ] pf_driver
  - [ ] package.xml and CMakeLists.txt

### CODE REVIEW

**Comment by ipa-hsd** [comment]
@wawanbreton I have added a very basic TODOs to the main description of the PR. Feel free to add to the list. Also let's try to make sure that the commits are created according to the list (e.g. one commit to port the `package.xml` and `CMakeLists.txt`)

**Comment by ipa-hsd** [comment]
@wawanbreton do you see a checkbox on this page that says 
```
Allow edits from maintainers
```
If yes, please check that box.

**Comment by wawanbreton** [comment]
Nice, I will take some time to update the TODO list.
I don't see the checkbox you mention, maybe it is on the branch page ? I will take a look.

**Comment by wawanbreton** [comment]
Actually, it seems that I have no right on this PR, as you are its author. So I can't see how to edit the description, neither to give you the commit permission. Sorry I'm quite new with workflows on Github !

**Comment by ipa-hsd** [comment]
@wawanbreton can you try to push your commits to https://github.com/PepperlFuchs/pf_lidar_ros_driver/pull/83? I could not find a solution to push to this PR.

File: pf_description/CMakeLists.txt [hunk 1]
```diff
@@ -1,27 +1,15 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.7)
```
Comment (ipa-hsd) [inline pf_description/CMakeLists.txt:1]: @wawanbreton this package has no cpp code so CMakeLists.txt is not needed. Please take a look at https://github.com/PepperlFuchs/pf_lidar_ros_driver/pull/83/commits/9476ad63b870801110c965d7b480135093a04f58 in https://github.com/PepperlFuchs/pf_lidar_ros_driver/pull/83

Comment (wawanbreton) [reply pf_description/CMakeLists.txt:1]: Isn't it required to properly deploy the target files ?

Comment (ipa-hsd) [reply pf_description/CMakeLists.txt:1]: what do you mean? for python only packages, there is a [setup.py](https://github.com/PepperlFuchs/pf_lidar_ros_driver/blob/porting-ros2/pf_description/setup.py)

Comment (wawanbreton) [reply pf_description/CMakeLists.txt:1]: But there is no Python code either :D so my answer is yes, something is required, either a CMakeLists.txt or a setup.py, which contains the deployment instructions

Comment (ipa-hsd) [reply pf_description/CMakeLists.txt:1]: launch files ;)

Comment (wawanbreton) [reply pf_description/CMakeLists.txt:1]: Indeed, but they can still be deployed with a CMakeLists.txt, although this is quite nonsense :)

**Comment by ipa-hsd** [comment]
closing in favour of https://github.com/PepperlFuchs/pf_lidar_ros_driver/pull/83

**Pull request closed without merging**